.gitignore

```diff
@@ -0,0 +1,2 @@
+/.idea/
+node_modules
```

Comment:
There is already a `.gitignore` in the client part of the repo. We can merge these two gitignores to one at the root of the repo.